sdk/parents/clientcore-io-parent/pom.xml

```diff
@@ -0,0 +1,1816 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.clientcore</groupId>
+  <artifactId>clientcore-io-parent</artifactId>
+  <packaging>pom</packaging>
+  <version>1.0.0</version> <!-- {x-version-update;io.clientcore:clientcore-io-parent;current} -->
```

Comment:
Let's use `1.0.0-beta.1` until we are ready to GA.

Reply:
updated.